app/Filament/App/Resources/LocalIndicatorResource.php

```diff
@@ -32,7 +32,14 @@ public static function form(Form $form): Form
                     ->options(Theme::all()->pluck('displayName', 'id'))
                     ->required()
                     ->preload()
-                    ->searchable(),
+                    ->searchable()
+                    ->columnSpanFull(),
+                Forms\Components\Select::make('global_indicator_id')
+                    ->relationship('globalIndicator', 'name')
+                    ->required()
```

Comment:
This should not be required; remember the workflow: 

1. Teams add the local indicators that come out of the LISP offline workshops.
2. Teams review the global indicators and see if any of the local indicators match. 

With this required, we can't have this 1-2 step-by-step.